fixtures/martin/quantum-wash.json

```diff
@@ -0,0 +1,177 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "Quantum Wash",
+  "categories": ["Moving Head", "Color Changer"],
+  "meta": {
+    "authors": ["Ryan Brady"],
+    "createDate": "2023-12-19",
+    "lastModifyDate": "2023-12-19"
+  },
+  "links": {
+    "manual": [
+      "https://www.martin.com/en-US/products/mac-quantum-wash"
```

Comment:
This is a product page.  The manual is at: https://www.martin.com/en-US/site_elements/martin-manuals-mac-quantum-wash-user-guide